packages/trace-viewer/src/ui/timeline.tsx

```diff
@@ -50,6 +51,10 @@ export const Timeline: React.FunctionComponent<{
 
   const { boundaries, offsets } = React.useMemo(() => {
     const boundaries = { minimum: model?.startTime || 0, maximum: model?.endTime || 30000 };
+    if (boundaries.minimum > boundaries.maximum) {
```

Comment:
How is this possible?

Reply:
Initial values min is maxint, max is minint.